### PR TITLE
model validations.

### PR DESCRIPTION
**Pull request by saratkavuru**

### CODE REVIEW

File: library_app/app/models/library.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
 class Library < ApplicationRecord
-  has_many :room
+  has_many :rooms
+  validates :library_id, :presence => true
```
Comment (goeltanmay) [inline library_app/app/models/library.rb:3]: shouldn't it be id instead of library_id?

File: library_app/app/models/room.rb [hunk 2]
```diff
@@ -1,3 +1,8 @@
 class Room < ApplicationRecord
   belongs_to :library
+  validates :room_number, :presence => true
+  validates_numericality_of :capacity, :message=>"Room size should be numerical"
```
Comment (goeltanmay) [inline library_app/app/models/room.rb:4]: isn't this field already numeric in database? do we need this?

File: library_app/app/models/room.rb [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+  validates :library_id, :presence => true
+
+
 end
```
Comment (goeltanmay) [inline library_app/app/models/room.rb:8]: Migration file?

**Review by goeltanmay — CHANGES_REQUESTED** [review]

File: library_app/app/models/room.rb (hunk 2 above)
Comment (saratkavuru) [reply library_app/app/models/room.rb:4]: Yeah this field is numeric. Even plain validates would do I guess

File: library_app/app/models/library.rb (hunk 1 above)
Comment (saratkavuru) [reply library_app/app/models/library.rb:3]: just checked the libraries table .. I think it should be :name

File: library_app/app/models/room.rb (hunk 3 above)
Comment (saratkavuru) [reply library_app/app/models/room.rb:8]: Wanted to be sure before I did the db migrate

**Pull request closed without merging**